base/constants.go

```diff
@@ -60,6 +60,7 @@ const (
 
 	DefaultViewQueryPageSize = 5000 // This must be greater than 1, or the code won't work due to windowing method
 
+	FilterByChannel = "sync_gateway/bychannel"
```

Comment:
If we're going to make this a constant, we should update all the other existing usages (in changes_api, admin_api).

Reply:
I'm just going to rollback this change, since there are so many places we could be using constants instead of strings, if we decide to do this, it probably makes more sense to do this comprehensively in a separate pass.